src/main/java/uk/gov/digital/ho/pttg/application/namematching/candidates/AbbreviatedNamesFunctions.java

```diff
@@ -13,24 +13,34 @@
 
     private static final String ANY_LETTER_INCLUDING_UNICODE_MATCHER = "\\p{L}\\p{M}*+";
     private static final String FULL_STOP_SPACE_MATCHER = "\\.\\s+";
+    private static final String APOSTROPHE_SPACE_MATCHER = "\'\\s+";
```

Comment:
This regex is wrong. Apostrophes have no special meaning in regular expressions so you don't need to escape them. The FULL_STOP_SPACE_MATCHER has two backslashes because in Java you have to escape those so you effectively end up with double.

If possible, don't just change it, work out what test case is missing that would highlight this error and add the test first.

Reply:
Resolved.